tools/exporters/DB_dump_table_data.php

```diff
@@ -56,7 +56,8 @@
 
 // Loop through all tables to generate insert statements for each.
 foreach ($tableNames as $tableName) {
-    $filename = __DIR__ . "/../../raisinbread/RB_files/RB_$tableName.sql";
+    $filename = \NDB_Factory::singleton()->settings()->getBaseURL()
```

Comment:
@johnsaigle doesnt baseurl return something like "abou-haider-dev.loris.ca" ?? thats not a path

Reply:
You're right, my bad. It should be project root or whatever the setting is called

Reply:
i think thats base